kickstart/etc/network/interfaces.d/wlan.cfg

```diff
@@ -0,0 +1,5 @@
+auto {{posm_wlan_netif}}
```

Comment:
Changing this to `allow-hotplug` will facilitate quicker booting when changing out wifi cards after installation and before fixing up `/etc/udev/rules.d/70-persistent-net.rules` (it will attempt to configure the previous interface, still on `wlan0`, while ignoring the new one, on `wlan1` until fixed).

This is enough of an edge case that people should only run into it when they're making changes. Plus, it makes a good reminder that the udev rules need to be fixed before things will work normally again.